sites/fast-website/src/app/components/content-placement-container/content-placement-container.template.ts

```diff
@@ -23,26 +23,28 @@ export const ContentPlacementContainerTemplate = html<ContentPlacementContainer>
         ${when(
             x => x.section === "feature",
             html<ContentPlacementContainer>`
-                ${repeat(
-                    x => x.featureCardData,
-                    html`<site-feature-card>
-                        <div>${x => x.item}</div>
-                        <h4>${x => x.header}</h4>
-                        <p slot="body">${x => x.body}</p>
-                        <fast-anchor
-                            slot="footer"
-                            href=${x => x.githubLink}
-                            appearance="lightweight"
-                            >View Github</fast-anchor
-                        >
-                        <fast-anchor
-                            slot="footer"
-                            href=${x => x.documentationLink}
-                            appearance="lightweight"
-                            >Read Documentation</fast-anchor
-                        >
-                    </site-feature-card>`
-                )}
+                <site-card-section>
```

Comment:
site-card-section should not need to be added here. Part of the purpose of moving feature-card here was to be able to use content-placement-container to style it. In my PR that you have rebased with I moved all the current styling of site-card-section into content-placement-container's styles. If there are updates to site-card-section they should be made in content-placement-container.style. It is redundant to apply site-card-section on top because content-placement-container should have the styles of site-card-section.

Reply:
Without it there, features is a single column.

Reply:
It looks as it should in feature/fast-frame, and feature/fast-frame does not use site-card-section. I am thinking that this is an issue with the rebase.

Reply:
Yeah when looking through your code it looks like you deleted where the flow variables were defined.